.obs/workflows.yml

```diff
@@ -65,3 +65,60 @@ workflow:
         source_project: home:defolos:BCI:CR:SLE-15-SP3
         source_package: micro
         target_project: home:defolos:BCI:CR:SLE-15-SP3:Staging
+  filters:
+    event: pull_request
+
+refresh_devel_BCI:
+  steps:
+    - trigger_services:
+        project: devel:BCI:SLE-15-SP3
+        package: python-3.6
+    - trigger_services:
+        project: devel:BCI:SLE-15-SP3
+        package: python-3.9
+    - trigger_services:
+        project: devel:BCI:SLE-15-SP3
+        package: rmt-nginx
+    - trigger_services:
+        project: devel:BCI:SLE-15-SP3
+        package: pcp-image
+    - trigger_services:
+        project: devel:BCI:SLE-15-SP3
+        package: rmt-server
+    - trigger_services:
+        project: devel:BCI:SLE-15-SP3
+        package: golang-1.18
+    - trigger_services:
+        project: devel:BCI:SLE-15-SP3
+        package: ruby-2.5-image
```

Comment:
that should be triggered against SP4, no?

Reply:
No, this is just telling obs to pull from git. What we do with the package result is not related to this.

Reply:
where can we remove those containers from the  sle15-sp3 branch?

Reply:
Don't create any `BaseContainerImage` instances for 15 SP3 anymore and `git rm` everything that you don't like from the `sle15-sp3` branch.